Release notes 6.0.21-18

nastena1606 opened the pull request:
Fixed broken links

modified:   docs/audit-logging.md
	modified:   docs/authentication.md
	modified:   docs/comparison.md
	modified:   docs/enable-auth.md
	modified:   docs/install/minor-upgrade.md
	modified:   docs/install/source.md
	modified:   docs/install/upgrade-from-mongodb.md
	modified:   docs/release_notes/6.0.2-1.md
	new file:   docs/release_notes/6.0.21-18.md
	modified:   docs/release_notes/index.md
	modified:   docs/x509-ldap.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

nastena1606 committed:
Release notes 6.0.21-18

Fixed broken links

modified:   docs/audit-logging.md
	modified:   docs/authentication.md
	modified:   docs/comparison.md
	modified:   docs/enable-auth.md
	modified:   docs/install/minor-upgrade.md
	modified:   docs/install/source.md
	modified:   docs/install/upgrade-from-mongodb.md
	modified:   docs/release_notes/6.0.2-1.md
	new file:   docs/release_notes/6.0.21-18.md
	modified:   docs/release_notes/index.md
	modified:   docs/x509-ldap.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

igorsol reviewed (approved):
LGTM

radoslawszulgo commented on docs/release_notes/6.0.21-18.md:
```diff
@@ -0,0 +1,63 @@
+
+## Release Highlights
+
+### Improved compatibility for data-at-rest encryption using KMIP between Percona Server for MongoDB with MongoDB Enterprise Edition 
```
```suggestion
### Improved compatibility for data-at-rest encryption using KMIP between Percona Server for MongoDB with MongoDB Enterprise Advanced 
```

radoslawszulgo commented on docs/release_notes/6.0.21-18.md:
```diff
@@ -0,0 +1,63 @@
+
+### Upstream Improvements
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
```
```suggestion
The bug fixes, provided by MongoDB Community Edition and included in Percona Server for MongoDB, are the following:
```

radoslawszulgo reviewed (commented):
As customers were asking - please include also following issues were fixed:
https://jira.mongodb.org/browse/SERVER-97842

nastena1606 committed:
updated after the review

nastena1606 committed:
Updated wording in changelog

nastena1606 committed:
Updated highlights:

Added Amazon Linux compatibiity info
Debug symbols as Pro build
Updated install and update docs with AL compatibility
Updated Pro features list

radoslawszulgo commented on snippets/al-compatibility.md:
```diff
@@ -0,0 +1,10 @@
+| Percona Server for MongoDB version | Amazon Linux 2023 version |
+|------------------------------------|---------------------------|
+| 6.0.20-17                          | 2023.6.x and older        |
+| 6.0.21-18                          | 2023.7.x            |
```
```suggestion
| 6.0.21-18                          | 2023.7.x and later            |
```

radoslawszulgo commented on snippets/al-compatibility.md:
```diff
@@ -0,0 +1,10 @@
+
+| Percona Server for MongoDB version | Amazon Linux 2023 version |
+|------------------------------------|---------------------------|
+| 6.0.20-17                          | 2023.6.x and older        |
```
```suggestion
| 6.0.20-17                          | 2023.6.x and earlier        |
```

nastena1606 committed:
Reworked Binary announcement, changed mongosh version

nastena1606 committed:
Update psmdb-pro.md

sandraromanchenko commented on mkdocs-base.yml:
```diff
@@ -139,7 +148,7 @@ plugins:
           output_path: '_pdf/PerconaServerforMongoDB-6.0.pdf'
           cover_title: 'Server for MongoDB Documentation 6.0'
-          cover_subtitle: 6.0.20-17 (February 19, 2025)
+          cover_subtitle: 6.0.21-18 (April 21, 2025)
```
```suggestion
          cover_subtitle: 6.0.21-18 (April 22, 2025)
```

nastena1606 committed:
Update snippets/al-compatibility.md

Co-authored-by: radoslawszulgo <radoslaw.szulgo@percona.com>

nastena1606 committed:
Update mkdocs-base.yml

Co-authored-by: Sandra Romanchenko <53295797+sandraromanchenko@users.noreply.github.com>
